src/core/mantle/evaluator/simple/expression/variableReplacer.ts

```diff
@@ -34,15 +34,15 @@ export function replaceVariables(expression: string, feature?: any): [string, JP
             return [result, []];
           }
         }
-        if (res.length == 1) {
+        if (res.length !== 0) {
           const placeholderLiteral = generateRandomString(10);
           literalJP.push({
             literalName: placeholderLiteral,
             literalValue: res[0],
           });
           result += placeholderLiteral;
         } else {
-          return [result, []];
+          return ["false", []];
```

Comment:
Returning result stops evaluating other condition, so I fixed to return false.

Reply:
Agreed.